### PR TITLE
Catch the ball

### LINKED ISSUE #4124: [New game]: Catch The ball
### 🎮 Game Request

Description:
Catch the Ball is an engaging and addictive arcade game that challenges players to catch a bouncing ball with a paddle.

### Point down the features

Key Features:

1)Intuitive Gameplay:

Simple controls: Move the paddle left or right to catch the ball.
Easy to understand, making it accessible for players of all skill levels.

Responsive Design:
Mobile-friendly: Optimized for touch controls on smartphones and tablets.
Resizable canvas: Automatically adjusts to different screen sizes, providing a consistent gaming experience on any device.

Endless Challenge:
Increasing difficulty: The ball's speed increases over time, testing players' reflexes and coordination.
Score tracking: Players can see their score in real-time and aim to beat their high score.
Screenshot 2024-06-02 132343

### Select program in which you are contributing

GSSoC24

### Code of Conduct

- [X] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) of this project.

### PR DESCRIPTION
**Pull request by shruti110503**
## PR Description 📜

Created the game catch the ball successfully.
Fixes #4124 

<hr>
 
## Mark the task you have completed ✅

- [x ] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) & [CODE OF CONDUCT](https://github.com/kunjgit/GameZone/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [ x] I have performed a self-review of my own code or work.
- [x ] I have commented my code, particularly in hard-to-understand areas.
- [ x] My changes generates no new warnings.
- [x ] I have followed proper naming convention showed in [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md)
- [x ] I have added screenshot for website preview in assets/images 
- [x ] I have added entries for my game in main README.md
- [ x] I have added README.md in my folder 
- [ x] I have specified the respective issue number for which I have requested the new game.

<hr>

## Add your screenshots(Optional) 📸

catch-the-ball

--- 
<br>

## Thank you soo much for contributing to our repository 💗

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -301,6 +301,9 @@ This repository also provides one such platforms where contributers come over an
 | [Pixel Painter](https://github.com/kunjgit/GameZone/tree/main/Games/pixel_painter)                                 |
 | [Guess_The_Song](https://github.com/kunjgit/GameZone/tree/main/Games/Guess_The_Song)                   | 
 | [NewsJunction](https://github.com/kunjgit/GameZone/tree/main/Games/NewsJunction)                  |
+
+
+=======
```
Comment (ishita-43) [inline README.md:306]: remove this line

File: README.md [hunk 2]
```diff
@@ -322,6 +325,7 @@ This repository also provides one such platforms where contributers come over an
 | [Tic-Tac-Toe Game](https://github.com/kunjgit/GameZone/tree/main/Games/Tic-Tac-Toe) |
 | [Rapid_click_frenzy](https://github.com/kunjgit/GameZone/tree/main/Games/Rapid_click_frenzy)                  
 
+| [Catch_The_Ball] (Games\Catch_The_Ball)                                 |
```
Comment (ishita-43) [inline README.md:328]: Enter your game properly 
refer other entries

**Review by ishita-43 — CHANGES_REQUESTED** [review]

**Comment by shruti110503** [comment]
@ishita-43 i have made the changes. Please review it.

File: README.md [hunk 3]
```diff
@@ -322,6 +322,7 @@ This repository also provides one such platforms where contributers come over an
 | [Tic-Tac-Toe Game](https://github.com/kunjgit/GameZone/tree/main/Games/Tic-Tac-Toe) |
 | [Rapid_click_frenzy](https://github.com/kunjgit/GameZone/tree/main/Games/Rapid_click_frenzy)                  
 
+| [Catch_The_Ball] (http://127.0.0.1:5501/Games/Catch_The_Ball/index.html)                              |
```
Comment (ishita-43) [inline README.md:325]: Place name of the game in the end instead!!
eg catch_the_ball should be same the folder name

**Review by ishita-43 — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (shruti110503) [reply README.md]: @ishita-43 please check if it is alright now

**Review by ishita-43 — APPROVED** [review]
Ready to merge!!

**Comment by kunjgit** [comment]
Thank you  @shruti110503 , for your valuable time and contribution in our GameZone 💗. 
 It’s our GameZone, so Let’s build this GameZone altogether !!🤝
  Hoping to see you soon with another PR again 😇 
 Wishing you all the best for your journey into Open Source🚀

**Pull request merged**